CIP-1694/README.md

```diff
@@ -420,22 +432,128 @@ However, this choice also limits the implementation effort for wallet providers
 
 Hence, this proposal promotes hard fork initiations as a standalone governance action, separate from protocol parameter updates, as in the Shelley scheme.
 
 ### Treasury Withdrawals vs Project Catalyst
 
-Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one treasury withdrawal per epoch.
 
 Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
 
+### The purpose of the DReps
+
+Nothing in this proposal limits the SPOs from becoming DReps.
+Why do we have DReps at all?
+SPOs are chosen purely for block production.
+Voters can choose to delegate their vote to DReps without needing to consider whether they are
+also a good block producer.
+
+### AVST (action voting stake threshold)
+
+The AVST exists to ensure that the votes of the DReps are meaningful.
+For example, if only 10 Lovelace were properly delegated to DReps, and 9 Lovelace "vote yes" to
+a governance action, we would doubt the legitimacy of the result since there are
+billions of ADA in circulation.
+A high enough AVST legitimizes the representation.
+
+### Ratification Requirements Table
+
+The requirements in the [ratification requirement table](#requirements) are explained here.
+Most of the governance actions have the same kind of requirements:
+the constitutional committee must reach quorum and the DReps must reach a sufficient number of
+'yes' votes, except in the situation where the AVST is too low, in which case the SPOs vote in
+place of the DReps.
+This includes these actions:
+* New Committee/quorum (normal state)
+* Update to the Constitution
+* Protocol parameter changes
+* Treasury withdrawal
+
+#### Motion of no-confidence
+
+A motion of no-confidence represents the lack of confidence by the Cardano community in the
+current Constitutional Committee, and hence the Constitutional Committee should have no
+say whatsoever in this governance action.
+In this situation, the SPOs and the DReps are left to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
```

Comment:
"Since the point is for both the DReps and SPOs combined to oust the CC, there is no fallback in case the DRep vote fails due to missing the AVST."

Similarly for the others - this is also an important choice that should be justified!

Reply:
I agree completely, the details around "we need to the DReps to vote, but haven't met AVST" are not nailed down yet with the researchers.